app/assets/asset-symbols/symbols.js

```diff
@@ -41,37 +41,39 @@ require("file-loader?name=asset-symbols/[name].png!./hero.png");
 require("file-loader?name=asset-symbols/[name].png!./ruble.png");
 require("file-loader?name=asset-symbols/[name].png!./oct.png");
 require("file-loader?name=asset-symbols/[name].png!./smoke.png");
-require("file-loader?name=asset-symbols/[name].png!./muse.png");
 require("file-loader?name=asset-symbols/[name].png!./ppy.png");
 require("file-loader?name=asset-symbols/[name].png!./stealth.png");
 require("file-loader?name=asset-symbols/[name].png!./kexcoin.png");
 require("file-loader?name=asset-symbols/[name].png!./bto.png");
 require("file-loader?name=asset-symbols/[name].png!./btm.png");
-require("file-loader?name=asset-symbols/[name].png!./krm.png");
-require("file-loader?name=asset-symbols/[name].png!./golos.png");
-require("file-loader?name=asset-symbols/[name].png!./gbg.png");
 require("file-loader?name=asset-symbols/[name].png!./atn.png");
 require("file-loader?name=asset-symbols/[name].png!./neo.png");
 require("file-loader?name=asset-symbols/[name].png!./gas.png");
 require("file-loader?name=asset-symbols/[name].png!./qtum.png");
 require("file-loader?name=asset-symbols/[name].png!./bkbt.png");
 require("file-loader?name=asset-symbols/[name].png!./dht.png");
 require("file-loader?name=asset-symbols/[name].png!./gxs.png");
-require("file-loader?name=asset-symbols/[name].png!./tt.png");
 require("file-loader?name=asset-symbols/[name].png!./scr.png");
 require("file-loader?name=asset-symbols/[name].png!./zeph.png");
 require("file-loader?name=asset-symbols/[name].png!./egem.png");
 require("file-loader?name=asset-symbols/[name].png!./hertz.png");
-require("file-loader?name=asset-symbols/[name].png!./dgb.png");
 require("file-loader?name=asset-symbols/[name].png!./doge.png");
 require("file-loader?name=asset-symbols/[name].png!./tusd.png");
 require("file-loader?name=asset-symbols/[name].png!./zec.png");
 require("file-loader?name=asset-symbols/[name].png!./waves.png");
 require("file-loader?name=asset-symbols/[name].png!./zrx.png");
 require("file-loader?name=asset-symbols/[name].png!./xrp.png");
 require("file-loader?name=asset-symbols/[name].png!./xmr.png");
-require("file-loader?name=asset-symbols/[name].png!./sbd.png");
 require("file-loader?name=asset-symbols/[name].png!./rudex.btc.png");
+require("file-loader?name=asset-symbols/[name].png!./rudex.eos.png");
```

Comment:
You have been renaming couple of assets here whilst adding your prefix. 

Do all of those assets have a special visual mark that makes them recognizable as rudex? Furthermore, have you checked that no other gateway is using the non-prefixed symbols?

Reply:
@ruslansalikhov ping